distribution/src/config/jvm.options

```diff
@@ -42,7 +42,6 @@
 # following three lines to your version of the JDK
```

Comment:
Drive by comment (sorry!), but the comment in `jvm.options` would be incorrect after this change, since the user would no longer be uncommenting the lines to use G1

Reply:
hmm, I wonder what we want to add instead.. we want to users to be able to use G1GC before jdk 14, but if they specifically add -XX:+UseG1GC the ergonomics choice won't kick in..

Reply:
Isn't the point fo the change to make ergonomics choose which GC to use? It doesn't just have to be based on small heaps, it can be based on java version as well?

Reply:
I updated the jvm.options, any comments/thoughts are welcomed!